lib/gollum-lib/filter/critic_markup.rb

```diff
@@ -22,7 +22,7 @@ class Gollum::Filter::CriticMarkup < Gollum::Filter
   def extract(data)
     data.gsub! ADDITION_PATTERN do
       content = $~[:content]
-      placeholder = "CRITIC" + Digest::SHA1.hexdigest("#{content}#{@map.size+1}")
+      placeholder = "CRITIC" + Digest::SHA1.hexdigest("#{content}#{@map.size}")
```

Comment:
Still had the question: why do we need to add even `@map.size`? If we have two identical bits of critic markup which generate the same hash, we don't have to store them under different placeholder keys, right?

Reply:
Right, because it might be a different kind of annotation. 👍

Reply:
Exactly. 👍